scripts/tests/test_validate_docstrings.py

```diff
@@ -60,9 +62,9 @@ def random_letters(self):
         letters : str
             String of random letters.
         """
-        length = random.randint(1, 10)  # noqa: F821
-        letters = ''.join(random.choice(string.ascii_lowercase)  # noqa: F821
-                          for i in range(length))
+        length = random.randint(1, 10)
+        letters = "".join(random.choice(string.ascii_lowercase)
+                          for _ in range(length))
```

Comment:
I know this comes from the original code, but we don't need a comprehension here, we can simply do `''.join(random.sample(string.ascii_lowercase, length))`

Reply:
Fair enough.  Done.